shared/team-building/filtered-service-tab-bar.tsx

```diff
@@ -9,7 +9,7 @@ export const FilteredServiceTabBar = (
   }
 ) => {
   const {selectedService, onChangeService} = props
-  const {servicesShown, minimalBorder, offset, filterServices} = props
```

Comment:
we removed all this offset logic awhile ago but never actually removed the props so it was always 0